packages/client/src/base-client.ts

```diff
@@ -1,3 +1,5 @@
+// Keep this around until we drop support for Node 14.
+import 'abort-controller/polyfill'; // eslint-disable-line import/no-unassigned-import
```

Comment:
I don't think that polyfill is actually making any difference. `AbortController` was added in 14.17, and we only support 14.18+. And based on that polyfill's [source code](https://github.com/mysticatea/abort-controller/blob/master/polyfill.js), it appears that the polyfill is a noop if that class already exists.